component/notification/src/main/java/org/exoplatform/social/notification/impl/SpaceWebNotificationServiceImpl.java

```diff
@@ -208,6 +222,20 @@ public Map<String, Long> countUnreadItemsByApplication(long userIdentityId, long
     return metadataService.countMetadataItemsByMetadataTypeAndAudienceId(METADATA_TYPE_NAME, userIdentityId, spaceId);
   }
 
+  @Override
+  public Map<Long, Long> countUnreadItemsBySpace(String authenticatedUser) {
+    List<Long> spaceIds = new ArrayList<>();
+    IdentityManager identityManager = getIdentityManager();
+    Identity userIdentity = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, authenticatedUser);
```

Comment:
Use a more simple method API without having to add the providerId

```suggestion
    Identity userIdentity = identityManager.getOrCreateUserIdentity(username);
    if (userIdentity == null) {
      throw new IllegalArgumentException("User " + username + " doesn't exist");
    }
```